getting-started/kubernetes/customizations/imageset.md

```diff
@@ -0,0 +1,199 @@
+
+#### Image digest
+
+Docker images, when added to a docker repository, have a unique hash created that can be used to pull a specific version of an image that cannot be changed or updated.
+
+### FAQ
+
+#### What do I need to do to prepare to upgrade my cluster?
+
+When upgrading to a new patch, minor, or major release it will be necessary to create a new [ImageSet]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.ImageSet)
+that contains updated image references for the new release and named for the new release. This should be done prior
+to upgrading the cluster so when the new manifests are applied the appropriate ImageSet is available.
+
+#### Why does the Installation Resource status not include my ImageSet?
+
+The [status.imageset]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.InstallationStatus) field of
+the Installation Resource will not be updated until the `calico` component has fully been deployed. `calico` is
+fully deployed when `kubectl get tigerastatus calico` reports Available True with Progressing and Degraded as False.
+
+#### How can I tell if there is a problem with my ImageSet?
+
+If you suspect an issue with your ImageSet, check tigerastatus with `kubectl get tigerastatus`. If any components are
+degraded you can get additional information with `kubectl get tigerastatus <component-name> -o yaml`. If the digest
+provided for an image is incorrect or unable to be pulled, the tigerastatus will not directly report that information
+but you should see information that there is an issue rolling out a Deployment, Daemonset, or Job. If you suspect
+an issue with a resource rollout due to an issue with an image you will need to `get` or `describe` a specific pod
+to see details about the problem.
+
+### Before you begin
+
+- To collect the information for an ImageSet you must have a docker client configured so it can pull images from the Docker repositories where the images are stored. 
+- Once you have created an ImageSet manifest you must have permissions to apply the manifest to your cluster, so you'll need the appropriate Kubernetes configuration.
+
+### How to
+
+- [Update the operator deployment with a digest](#update-the-operator-deployment-with-a-digest)
+- [Create an ImageSet manually](#create-an-imageset-manifest) or [Create an ImageSet with a script](#section-below)
+
+#### Verification steps
+
+1. Check tigerastatus for components that are Degraded with `kubectl get tigerastatus`.
+   - If any components show Degraded, [investigate further](#how-can-i-tell-if-there-is-a-problem-with-my-imageset).
+2. When tigerastatus for all components show Available True, the ImageSet has been applied.
+   ```
+   NAME     AVAILABLE   PROGRESSING   DEGRADED   SINCE
+   calico   True        False         False      54s
+   ```
+3. Verify that the correct ImageSet is being used by checking the `imageset` field in the Installation status is set to the ImageSet you created.
```

Comment:
Verify that the correct ImageSet is being used. In Installation status, check that the `imageset` field is set to the ImageSet you created.